sinks/cortex/cortex.go

```diff
@@ -13,6 +13,7 @@ import (
 
 	"github.com/golang/protobuf/proto"
 	"github.com/golang/snappy"
+	"github.com/hashicorp/go-multierror"
```

Comment:
No concern from me, but if anyone has concern about increasing the dependency surface area alternatives are listed here: https://stackoverflow.com/questions/33470649/combine-multiple-error-strings

Reply:
Oh nice! https://github.com/golang/go/issues/53435
1.20 (can't use yet)